diff --git a/LoopKit/LoopKit/Extensions/Guardrail+Settings.swift b/LoopKit/LoopKit/Extensions/Guardrail+Settings.swift
index 1c8398f..ed00ef6 100644
--- a/LoopKit/LoopKit/Extensions/Guardrail+Settings.swift
+++ b/LoopKit/LoopKit/Extensions/Guardrail+Settings.swift
@@ -81,3 +81,3 @@ public extension Guardrail where Value == HKQuantity {
     static let insulinSensitivity = Guardrail(
-        absoluteBounds: (9.1)...(500.9),
+        absoluteBounds: (9.1)...(1001.9),
         recommendedBounds: (15.1)...(399.9),
